### ISSUE TITLE
Animation load twice on page refresh in chrome

### ISSUE THREAD

**Issue by manektechmayank** [elementor/elementor#6945]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
I have created 3 images. And given animation "left", "right" and "bottom" accordingly. When I scroll the page it's working fine. But when refresh the page, then animation happen twice. Means it start the animation twice in Chrome browser.

## Steps to reproduce
1) Create any section using elementor plugin.
2) Add 3 image by using "image widget".
3) Then give the positioning of image using margin so that they will look like below image.

![image](https://user-images.githubusercontent.com/31854318/52123224-7347d180-264b-11e9-91c9-38634cd53a9d.png)

4) Now it will work fine on first time page load but If you refresh the page then it will look like start twice animation.

**Please check this video so that you can see clear issue of animation :** https://www.screencast.com/t/7IPLqrKym5Jx

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
					
== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.6 (CentOS) OpenSSL/1.0.2k-fips PHP/7.2.9
	MySQL version: MySQL Community Server (GPL) v5.7.12
	PHP Version: 7.2.9
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: No
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.0.3
	Site URL: http://localhost
	Home URL: http://localhost
	WP Multisite: No
	Max Upload Size: 2 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Elementor Hello Theme
	Version: 1.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.81 Safari/537.36

== Active Plugins ==
	BackupBuddy
		Version: 8.3.5.0
		Author: iThemes

	Classic Editor
		Version: 1.3
		Author: WordPress Contributors

	Custom Post Type UI
		Version: 1.6.1
		Author: WebDevStudios

	Duplicate Post
		Version: 3.2.2
		Author: Enrico Battocchi

	Elementor
		Version: 2.4.4
		Author: Elementor.com

	Elementor Extras
		Version: 2.0.8
		Author: Namogo

	Elementor Pro
		Version: 2.4.2
		Author: Elementor.com

	Elementor Siteset
		Version: 1.4.1
		Author: Elementor Siteset


== Log ==
	: 
Log: showing 2 of 22019-01-28 15:45:20 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.1',
  'to' => '2.4.4',
)]
2019-01-28 15:45:32 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.3.1',
  'to' => '2.4.2',
)]

JS: showing 2 of 2JS: 2019-01-28 20:24:49 [error X 3][http://localhost/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.4.4:2:6883] Cannot read property \'select2\' of undefined 
JS: 2019-02-01 04:40:07 [error X 5][http://localhost/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.4.4:2:74537] setting getter-only property \"type\" 


				

</details>

**Comment by shilo-ey** [comment]
Hi @manektechmayank 

Thanks for reporting.

We couldn't reproduce your issue (after recreating the same layout & attributes) and therefore we suggest you to try deactivating all of your plugins except Elementor & Elementor Pro, and see if it resolved.

thanks.

**Comment by wayheming** [comment]
Hello @shilo-ey I have the same problem https://gyazo.com/875cb468a88a0b47b1bd6baab14d7969

**Comment by wayheming** [comment]
I think the problem with slide animations.

**Comment by shilo-ey** [comment]
Hi @manektechmayank & @wayheming 

This issue has been fixed in the upcoming major version.

Thanks

**Comment by wayheming** [comment]
Hello @shilo-ey,
thanks for the quick response as always.

**Comment by shilo-ey** [comment]
This has been fixed in Elementor v2.5.0

Feel free to update!
